src/autobump.ts

```diff
@@ -2,38 +2,38 @@ import dotenv from 'dotenv';
 import dotenvExpand from 'dotenv-expand';
 import { getUnixTime } from 'date-fns';
 import { ThreadChannel } from 'discord.js';
+import { Result } from 'oxide.ts';
 import { getDiscordClient } from './clients';
 import { listAllThreads } from './commands/autobump-threads/util';
 
 const env = dotenv.config();
 dotenvExpand.expand(env);
 
 const autobump = async () => {
-  const settings = await listAllThreads();
-  if (!settings.success) {
+  const settings = await Result.safe(listAllThreads());
```

Comment:
This `await Result.safe()` from `oxide.ts` will make this into a safe promise. which can then be `unwrap()`ped to get the value out. See https://github.com/traverse1984/oxide.ts